vllm/attention/backends/hpu_attn.py

```diff
@@ -125,6 +125,8 @@ def __init__(
         kv_cache_dtype: str,
```

Comment:
please modify requirements-hpu.txt to point to apropriate vllm-hpu-extension commit (0766759), fix static code analisys issues and we can merge